### PR TITLE
Team

### PR DESCRIPTION
**Pull request by henryyang42**
Add app team which has 2 views
1. team_list
   - list all teams' info
   - support paging
2. team_profile
   - show team's statistic (profile/ joined contests)
   - show team member info
   - allow some modification like change leader/ kick member

### CODE REVIEW

File: nthuoj/urls.py [hunk 1]
```diff
@@ -3,6 +3,6 @@
 
 urlpatterns = patterns('',
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^submit/$', 'users.views.submit'),
-    url(r'^profile/$', 'users.views.profile'),
```
Comment (nickelcheng) [inline nthuoj/urls.py]: Why do you delete line 6 & 7?

Comment (henryyang42) [inline nthuoj/urls.py]: I include('users.urls') and add users/urls.py instead.

File: team/templates/team/team_profile.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
+<!--
```
Comment (nickelcheng) [inline team/templates/team/team_profile.html]: I suggest changing the filename to "teamProfle.html".

Comment (henryyang42) [inline team/templates/team/team_profile.html]: OK.
And so does teamList.html?

Comment (nickelcheng) [inline team/templates/team/team_profile.html]: Yes.

**Comment by bruce3557** [comment]
I think this PR is ok for me.
If @nickelcheng has no other comments about logic, we can merge this after you merge the latest code.

**Comment by nickelcheng** [comment]
I think it's OK.

**Comment by nickelcheng** [comment]
Merged, thanks @henryyang42 's effort!

**Pull request merged**